tracer/test/snapshots/Iast.StandaloneBilling.AspNetCore5.IastEnabled.verified.txt

```diff
@@ -0,0 +1,84 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/typereflectioninjection,
+    Service: Samples.Security.AspNetCore5,
+    Type: web,
+    Tags: {
+      aspnet_core.endpoint: Samples.Security.AspNetCore5.Controllers.IastController.TypeReflectionInjection (Samples.Security.AspNetCore5),
+      aspnet_core.route: iast/typereflectioninjection,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/typereflectioninjection,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/TypeReflectionInjection?type=System.String,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "REFLECTION_INJECTION",
+      "hash": -1084027092,
+      "location": {
+        "spanId": XXX,
+        "path": "Samples.Security.AspNetCore5.Controllers.IastController",
+        "method": "TypeReflectionInjection"
+      },
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "System.String",
+            "source": 0
+          }
+        ]
+      }
+    }
+  ],
+  "sources": [
+    {
+      "origin": "http.request.parameter",
+      "name": "type",
+      "value": "System.String"
+    }
+  ]
+}
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.limit_psr: 0.5,
+      _dd.rule_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: -1.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: aspnet_core_mvc.request,
+    Resource: GET /iast/typereflectioninjection,
+    Service: Samples.Security.AspNetCore5,
+    Type: web,
+    ParentId: Id_2,
+    Tags: {
+      aspnet_core.action: typereflectioninjection,
+      aspnet_core.controller: iast,
+      aspnet_core.route: iast/typereflectioninjection,
+      component: aspnet_core,
+      env: integration_tests,
+      language: dotnet,
+      span.kind: server
+    },
+    Metrics: {
+      _dd.apm.enabled: 0.0
```

Comment:
Just to confirm, this _only_ needs to go on the "first" span in the trace chunk right? It doesn't need to go on the _root_ span, or on _all_ spans?

Reply:
When APM is disabled, the numeric tag `_dd.apm.enabled:0` must be added to the metrics map of the first span of every trace chunk.